prow/cmd/deck/README.md

```diff
@@ -36,3 +36,10 @@ tar -xvf ./bazel-bin/prow/cmd/deck/spyglass-lenses-layer.tar -C /tmp/deck
 --static-files-location=/tmp/deck/static
 --spyglass-files-location=/tmp/deck/lenses
 ```
+
+## Rerun Prow Job via Prow UI
+
+Rerun prow job can be done by visting prow UI, locate prow job and rerun job by clicking on the ↻ button then clicking `Rerun` button. For prow on github, the permission is controlled by github membership, and configured as part of deck configuration, see [`rerun_auth_configs`](https://github.com/kubernetes/test-infra/blob/0dfe42533307f9733f22d4a6abf08e1df2229fcb/config/prow/config.yaml#L92) for k8s prow.
```

Comment:
We should probably indicate that this is available for non GitHub Prow's as well if the front end is secured and `allow_anyone` is set to true for the job: https://github.com/kubernetes/test-infra/blob/95cc9f4b68d0ce5702c3b3e009221de0fe0a482a/prow/apis/prowjobs/v1/types.go#L190-L191

Reply:
good point, done